x-pack/functionbeat/manager/aws/op_ensure_bucket.go

```diff
@@ -42,10 +44,22 @@ func (o *opEnsureBucket) Execute(_ executor.Context) error {
 	}
 
 	var apiError smithy.APIError
+	var input *s3.CreateBucketInput
 	if errors.As(err, &apiError) {
 		if apiError.ErrorCode() == notFound {
+			if o.region == "us-east-1" {
```

Comment:
I guess we don't have to use the if else here since when o.region=us-east-1, we can still just have `LocationConstraint` populated using `us-east-1` right?